lib/db.js

```diff
@@ -202,5 +202,32 @@ export default {
       const errorResponse = this.dynamoErrorResponse(err);
       throw errorResponse;
     }
+  },
+
+  updateDBCustom: async function(params) {
+    try {
+      const command = new UpdateCommand(params);
+      const res = await docClient.send(command);
+      return res;
+    } catch (err) {
+      const errorResponse = this.dynamoErrorResponse(err);
+      throw errorResponse;
+    }
+  },
+
+  put: async function(obj, table) {
```

Comment:
i'm just gonna keep this here – gonna test if put works with the .promise() code if not then we'll use this function